ibm_i/datadog_checks/ibm_i/query_script.py

```diff
@@ -0,0 +1,35 @@
+import sys
```

Comment:
I don't know where we should add that, but I think the check would benefit from having some developer docs explaining why we're fetching metrics in such a peculiar way (ie. explaining the limitations of the IBM i ODBC driver, why we need to maintain the connection across check runs, and how this solution works around it).

Reply:
@ofek what do you think is the best way to go about this?

Reply:
Great idea! Please write it here: https://github.com/DataDog/integrations-core/tree/master/docs/developer/architecture

https://datadoghq.dev/integrations-core/architecture/snmp/

Reply:
I added the docs on d622fbc66603efc5b2398ed506520e9f5236eeb5 PTAL @ofek @KSerrania